spec/queries/patient_states/monthly_registrations_query_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+describe PatientStates::MonthlyRegistrationsQuery do
+  around do |example|
+    with_reporting_time_zone { example.run }
+  end
+
+  let(:regions) { setup_district_with_facilities }
+  let(:period) { Period.current }
+
+  context "monthly registrations" do
+    it "returns the registrations in a facility for that month" do
+      facility_1_new_registrations = create_list(:patient, 2, assigned_facility: regions[:facility_1])
```

Comment:
Use different months as the value for device_created_at, for a more robust test.